crates/matrix-sdk-ui/src/timeline/event_item/mod.rs

```diff
@@ -442,6 +443,19 @@ impl EventTimelineItem {
         }
     }
 
+    pub fn should_boost(&self) -> bool {
+        match self.content() {
+            TimelineItemContent::Message(msg) => match msg.msgtype() {
+                MessageType::Text(text) => {
```

Comment:
What about other event types with captions?

Reply:
👍  added caption support for the Audio, File, Image and Video MessageTypes